flows/confidence-level-joining/events.json

```diff
@@ -0,0 +1,343 @@
+      "version": "1.0",
+      "time": 19000,
+      "domainId": "example.domain",
+      "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee12"
+    },
+    "data": {
+      "placeholder": "The syntax of this event has not yet been defined."
+    },
+    "links": {
+      "testCaseExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee8"
+    }
+  },
+  {
+    "meta": {
+      "type": "EiffelTestCaseFinishedEvent",
+      "version": "1.0",
+      "time": 14000,
+      "domainId": "example.domain",
+      "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee13"
+    },
+    "data": {
+      "placeholder": "The syntax of this event has not yet been defined."
+    },
+    "links": {
+      "testCaseExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee9"
+    }
+  },
+  {
+    "meta": {
+      "type": "EiffelTestCaseFinishedEvent",
+      "version": "1.0",
+      "time": 16000,
+      "domainId": "example.domain",
+      "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee14"
+    },
+    "data": {
+      "placeholder": "The syntax of this event has not yet been defined."
+    },
+    "links": {
+      "testCaseExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee10"
+    }
+  },
+  {
+    "meta": {
+      "type": "EiffelTestCaseFinishedEvent",
+      "version": "1.0",
+      "time": 15000,
+      "domainId": "example.domain",
+      "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee15"
+    },
+    "data": {
+      "placeholder": "The syntax of this event has not yet been defined."
+    },
+    "links": {
+      "testCaseExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee11"
+    }
+  },
+  {
+    "meta": {
+      "type": "EiffelActivityFinishedEvent",
+      "version": "1.0",
+      "time": 20000,
+      "domainId": "example.domain",
+      "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee16"
+    },
+    "data": {
+      "outcome": {
+        "conclusion": "UNSUCCESSFUL"
+      }
+    },
+    "links": {
+      "activityExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee4"
+    }
+  },
+  {
+    "meta": {
+      "type": "EiffelActivityFinishedEvent",
+      "version": "1.0",
+      "time": 20000,
+      "domainId": "example.domain",
+      "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee17"
+    },
+    "data": {
+      "outcome": {
+        "conclusion": "SUCCESS"
+      }
+    },
+    "links": {
+      "activityExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee5"
+    }
+  },
+  {
+    "meta": {
+      "type": "EiffelConfidenceLevelModifiedEvent",
+      "version": "1.0",
+      "time": 21000,
+      "domainId": "example.domain",
+      "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee18"
+    },
+    "data": {
+      "name": "functionalComponentTestsPassed",
+      "value": "FAILURE",
+      "issuer": {
+        "name": "componentTestsListener"
+      }
+    },
+    "links": {
+      "causes": [
+        "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee13",
+        "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee14",
+        "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee15",
+        "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee16"
+      ],
+	  "subjects": [
```

Comment:
Thanks! Those tabs have been annoying me to no end. I finally managed to change the editor settings, so it shouldn't be a problem anymore.